### PR TITLE
(DOCSP-22378) Release notes, source constants for v1.0.3

### PR DESCRIPTION
**Pull request by sarahsimpers**
[Build Link](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=626ad94f0fdfea9c37c3844b)
[Jira Ticket Link](https://jira.mongodb.org/browse/DOCSP-22378)
[Staging](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-22378/atlas-cli-changelog/#atlas-cli-1.0.3)

### CODE REVIEW

File: source/atlas-cli-changelog.txt [hunk 1]
```diff
@@ -12,6 +12,33 @@
...
+  - {+atlas-cli+} automatically identifies the ``mongosh`` path for
+    :ref:`atlas-config` and :ref:`atlas-quickstart` commands.
+
+- Makes minor improvements to the {+atlas-cli+} documentation.
+
+- Adds an alias ``-`` as the standard output for the parameter
+  ``--outFor`` for the following commands:
+
+  - :ref:`atlas-logs-decrypt`
+  - :ref:`atlas-serverUsage-download`
```
Comment (andreaangiolillo) [inline source/atlas-cli-changelog.txt:40]: @fmenezes Could you confirm this?

**Review by andreaangiolillo — COMMENTED** [review]
LGTM. Please wait for @[fmenezes](https://github.com/fmenezes) to close my comment.

File: source/atlas-cli-changelog.txt [hunk 2]
```diff
@@ -12,6 +12,33 @@
...
+    with the cluster prefix and five digits.
+  - :ref:`atlas-quickstart` now displays a default cluster
+    option before starting interactive setup.
+  - {+atlas-cli+} automatically identifies the ``mongosh`` path for
+    :ref:`atlas-config` and :ref:`atlas-quickstart` commands.
+
+- Makes minor improvements to the {+atlas-cli+} documentation.
+
+- Adds an alias ``-`` as the standard output for the parameter
+  ``--outFor`` for the following commands:
```
Comment (fmenezes) [inline source/atlas-cli-changelog.txt:37]: the name of the parameter is out
```suggestion
  ``--out`` for the following commands:
```

File: source/atlas-cli-changelog.txt [hunk 3]
```diff
@@ -12,6 +12,33 @@
...
+
+  - :ref:`atlas-logs-decrypt`
```
Comment (fmenezes) [inline source/atlas-cli-changelog.txt:39]:
```suggestion
  - :ref:`atlas-logs-download`
```

File: source/atlas-cli-changelog.txt (hunk 1 above)
Comment (fmenezes) [reply source/atlas-cli-changelog.txt:40]: @andreaangiolillo added comments

File: source/atlas-cli-changelog.txt [hunk 4]
```diff
@@ -12,6 +12,32 @@
...
+Released 28 April 2022
+
+Changes
+~~~~~~~
+
+- Improves :ref:`atlas-quickstart` commands:
+
+  - Clusters created using :ref:`atlas-quickstart` now start
+    with the cluster prefix and five digits.
+  - :ref:`atlas-quickstart` now displays a default cluster
```
Comment (zach-carr) [inline source/atlas-cli-changelog.txt:29]: [optional nit] Is this worth capitalizing as the start of the bullet point? "Atlas" isn't capitalized in the other links so I don't know.
```suggestion
  - :ref:`Atlas quickstart <atlas-quickstart>` now displays a default cluster
```

**Review by zach-carr — APPROVED** [review]
LGTM

**Comment by sarahsimpers** [comment]
@fmenezes When I tried to access the ref for `atlas-serverUsage-download` it does not resolve correctly, so I suspect that command should also be removed (in addition to the change you suggested that replaces `atlas-logs-decrypt`. I don't want to merge and publish this until I confirm that `atlas-serverUsage-download` should also be removed so please let me know. Thank you!

**Comment by fmenezes** [comment]
> @fmenezes When I tried to access the ref for `atlas-serverUsage-download` it does not resolve correctly, so I suspect that command should also be removed (in addition to the change you suggested that replaces `atlas-logs-decrypt`. I don't want to merge and publish this until I confirm that `atlas-serverUsage-download` should also be removed so please let me know. Thank you!

@sarahsimpers I can see the command in here https://www.mongodb.com/docs/mongocli/v1.16/reference/ops-manager/server-usage-download/ but looks like it is not in newer versions, I'll check later on.

**Comment by fmenezes** [comment]
Just checked server commands are available for ops manager but not atlas, so the command does not exist on atlas cli

**Pull request merged**